### PR TITLE
BZ2090176: Updates OpenShift Data Foundation version to 4.10 in OCP 4.10 doc

### PR DESCRIPTION
**Pull request by xenolinux**
Updates OpenShift Data Foundation version to 4.10 in OCP 4.10 doc

Version(s): 4.10 only

Issue: https://bugzilla.redhat.com/show_bug.cgi?id=2090176

Link to docs preview: http://file.rdu.redhat.com/sdudhgao/update-links/storage/persistent_storage/persistent-storage-ocs.html

Additional information:

### CODE REVIEW

**Review by jldohmann — APPROVED** [review]
/lgtm

File: storage/persistent_storage/persistent-storage-ocs.adoc [hunk 1]
```diff
@@ -8,7 +8,7 @@ toc::[]
 
 {rh-storage-first} is a provider of agnostic persistent storage for {product-title} supporting file, block, and object storage, either in-house or in hybrid clouds. As a Red Hat storage solution, {rh-storage-first} is completely integrated with {product-title} for deployment, management, and monitoring.
 
-{rh-storage-first} provides its own documentation library. The complete set of {rh-storage-first} documentation identified below is available at https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.9.
+{rh-storage-first} provides its own documentation library. The complete set of {rh-storage-first} documentation identified below is available at https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.10.
```
Comment (jeana-redhat) [inline storage/persistent_storage/persistent-storage-ocs.adoc:11]: I know this is just an update of existing text, but links should have friendly text. Suggestion:
```suggestion
{rh-storage-first} provides its own documentation library. The complete set of {rh-storage-first} documentation identified below is available in the link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.10[Product Documentation for Red Hat OpenShift Data Foundation 4.10].
```

**Review by jeana-redhat — COMMENTED** [review]
Updates LGTM. Suggest fixing this existing link issue while you're here (if you don't mind).

File: storage/persistent_storage/persistent-storage-ocs.adoc (hunk 1 above)
Comment (xenolinux) [reply storage/persistent_storage/persistent-storage-ocs.adoc]: Addressed

**Pull request merged**